package.json

```diff
@@ -8,8 +8,9 @@
     "lint-update": "curl 'https://raw.githubusercontent.com/apiaryio/coffeescript-style-guide/master/coffeelint.json' > coffeelint.json",
     "precompile": "npm run lint",
     "compile": "coffee -b -c -o lib/ src/",
-    "server-test": "mocha --compilers coffee:coffee-script/register -R spec --recursive --timeout 5000 test/**-test.coffee",
-    "test": "npm run lint && npm run server-test",
+    "test": "npm run lint && mocha",
```

Comment:
I've move all the mocha arguments out to the [`test/mocha.opts` file](http://mochajs.org/#mochaopts) so we can run `mocha` manually easier (for example to run a specific test).

```shell
$ mocha test/create-adapter-test.coffee 


  #createAdapter
    ✓ recognizes raw JSON serialization of API Blueprint AST
    ✓ recognizes JSON serialization of API Blueprint AST
    ✓ recognizes JSON serialization of API Blueprint AST
    ✓ recognizes API Blueprint AST
    ✓ does not recognize raw YAML serialization of API Blueprint AST
    ✓ does not recognize JSON serialization of API Blueprint AST with rendered HTML
    ✓ does not recognize YAML serialization of API Blueprint AST with rendered HTML
    ✓ does not recognize mime type with wrong base type
    ✓ recognizes JSON serialization of refract with json suffix
    ✓ recognizes JSON serialization of refract without suffix
    ✓ does not recognizes YAML serialization of refract with yaml suffix


  11 passing (13ms)
```

I've also omitted some unnecessary options such as `-R spec` since spec is the default reporter along specifying `test/**-test.coffee` since this is unnecessary. There is also no need to set the timeout to 5 seconds, the default 2 seconds such suffice and also prevent anyone making tests that take too long.